src/chaat/db/user.clj

```diff
@@ -0,0 +1,39 @@
+(ns chaat.db.user
+  (:require
+   [next.jdbc :as jdbc]
+   [next.jdbc.sql :as sql]))
+
+;; these 2 functions do the same thing, will remove one of them
```

Comment:
[nit] Same as a previous comment above. Let's prefer addressing these type of comments right away rather than keeping it around. Unless it's a large refactor.

Reply:
Addressed (:
Decided to keep both of them, but one is implemented using the other.